server/tests/api/test_reports.py

```diff
@@ -77,3 +77,35 @@ def test_jurisdiction_admin_report(
         == 'attachment; filename="audit-report-J1-Test-Audit-test-jurisdiction-admin-report-DATETIME.csv"'
     )
     assert_match_report(rv.data, snapshot)
+
+
+def test_report_before_audit_starts(
```

Comment:
just curious do we have something like jest coverage that tracks the test coverage in ARLO?

Reply:
yeah, it's configured in .coveragerc. currently at 99.6%. i'd like to get to 100% - would probably only take a few hours